buildenv

```diff
@@ -0,0 +1,45 @@
+export ZOPEN_STABLE_URL="git@github.com:reconquest/shdoc.git"
+export ZOPEN_STABLE_DEPS="bash make automake autoconf git zoslib gawk sed grep"
+export ZOPEN_BUILD_LINE="STABLE"
+export ZOPEN_RUNTIME_DEPS="bash gawk"
+
+
+export ZOPEN_BOOTSTRAP="skip"
+export ZOPEN_CONFIGURE="skip"
+export ZOPEN_BUILD="skip"
+export ZOPEN_CLONE_SUBMODULES="yes"
+export ZOPEN_INSTALL_OPTS="install DST=\${ZOPEN_INSTALL_DIR}/bin/"
+
+zopen_pre_install() {
+  mkdir -p "${ZOPEN_INSTALL_DIR}/bin/"
+}
+
+zopen_check_results()
+{
+  dir="$1"
+  pfx="$2"
+  chk="$1/$2_check.log"
+
+  # Echo the following information to gauge build health
```

Comment:
Can you add code to get actual failures here?

Reply:
I have added a task - https://github.com/ZOSOpenTools/shdocport/issues/3 , Needs gawk issue fix - https://github.com/ZOSOpenTools/gawkport/issues/22